notes/markov.md

```diff
@@ -76,7 +127,9 @@ A steady state vector \\({\bf x}^*\\) is a probability vector (entries are non-n
 <div>\[ {\bf M} {\bf x}^* = {\bf x}^* \]</div>
```

Comment:
you might want to more formally state the Markov property and note that the 1-norm of each column of a Markov matrix is always 1